### ISSUE TITLE
APA comment: align altText restrictions with HTML alt attribute

### ISSUE THREAD

**Issue by nigelmegitt** [w3c/imsc#321]
From https://lists.w3.org/Archives/Public/public-tt/2018Jan/0270.html

Section 7.7.4 itts:altText.  While we see this feature as useful for accessibility purposes, it should be mandatory for images rather than recommended only. As mentioned in the spec, one could take the pertaining text passage from the text profile document instance – but (1) an accompanying text profile is not required, and (2) the alternative text for the image could be different from the textual caption. Therefore, the itts:altText element should always be specified, but it should be empty for decorative images (not clear if a “decorative image” used as a caption makes sense anyway). By requiring an itts:altText for every image, but allowing for an empty element in case of a decorative image, we would align it with the alt attribute in HTML5 for images.

**Comment by palemieux** [comment]
`itts:altText` is maintained only for compatibility with IMSC1 and otherwise deprecated in favor of the TTML2 altText named metadata item. No further changes to `itts:altText` should therefore be made. An issue should be filed against the altText named metadata item as defined in TTML2, preferably, or as constrained in IMSC1.1 alternatively.

**Comment by nigelmegitt** [comment]
I'm not sure I understand the language in the comment:

> ... the itts:altText element should always be specified ...

then 

> By requiring an itts:altText for every image ...

I am not sure if this is asking for the presence of `itts:altText` to be required or recommended? 

It is already recommended in §9.4.3 and §9.4.4, for example:

> A `div` element that contains a child image element SHOULD contain a `metadata` element containing an `ittm:altText` element that is a Text Alternative of the image resource referenced by the `image` element.

Additionally appendix D. WCAG Considerations explicitly calls this out and strengthens that recommendation.

Unless the comment is requesting that this be a SHALL I think there is no action to take here.

**Comment by palemieux** [comment]
Proposed disposition: WR-rejected

**Comment by nigelmegitt** [comment]
@palemieux I think that's the correct disposition, however in my mind it is subject to any further clarification of my question at https://github.com/w3c/imsc/issues/321#issuecomment-361672359
